packages/test/mocha-test-setup/mocharc-common.cjs

```diff
@@ -8,6 +8,17 @@
 const { existsSync } = require("fs");
 const path = require("path");
 
+/**
+ * Get the mocha configuration for running tests using the conventions followed in the Fluid Framework repository.
+ *
+ * @param {string} packageDir - the directory of the package, typically set using `__dirname`
+ * @param {string[]} additionalRequiredModules - modules to require in addition to the standard set.
+ * @param {string} testReportPrefix - prefix for the test output report file names.
+ * @remarks
+ * Additional configuration can be provided via environment variables: see {@link file://./README.md}.
+ *
+ * Users desiring exact control over the `spec` from the CLI should delete or replace the spec from the returned config.
```

Comment:
I first wrote a comment asked if it was necessary, then saw your own comment talking about this, so suggesting to make it explicit here too.
```suggestion
 * Users desiring exact control over the `spec` from the CLI should delete or replace the spec from the returned config, since mocha's behavior is to extend it, not override it.
```